kubeflow/common/iap-secret/Makefile

```diff
@@ -11,4 +11,4 @@ iap-secret: check-iap
 
 .PHONY: check-iap
 check-iap:
-	../../hack/check_oauth_secret.sh
+	./check_oauth_secret.sh
```

Comment:
Can you clarify why this has to be a separate step than ingress-iap?

Reply:
After thinking about it, I merged the iap-secret to iap-ingress/ folder. I was wondering whether it makes sense to merge two into one: is there scenario where user wants to only update one folder, but not the other. I think this case is not very common, so sounds good on merging this iap secret to iap-ingress/ folder: https://github.com/kubeflow/gcp-blueprints/pull/238/files